study.md

```diff
@@ -49,19 +50,22 @@ unique. Write the migration would be used to create a table satisfying these
 requirements.
 
 ```ruby
-# your response here
+create_table :pets do |t|
+  t.string :name, :unique true, false
+  t.string :breed
 ```
 
 ## Explain the Role of Seed Data
 
 In your own words, explain the role of application seed data.
 
 ```md
-<!-- your response here -->
+Intialize the setup of the application. Please advise as to what the role
+is as I really am not sure.
 ```
 
 Should seed data be used for experimentation during development?
 
 ```md
-<!-- your response here -->
+No, the seed data should never be touched I believe.
```

Comment:
Seed data should be used during development since it can populate a database without having to manually enter data for testing.